README.md

```diff
@@ -1,50 +1,46 @@
-[![official JetBrains project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
-
-This repository contains sources for [try.kotl.in]( http://try.kotlinlang.org/)
-
-## Manual installation :whale:
-Before starting the Kotlin-Web-Demo execute two gradle tasks: `copyKotlinLibs` for downloading kotlin libraries for compiler and
-`war` for building war-archive from IDE or terminal:
-```bash
-$ gradlew ::copyKotlinLibs
-$ gradlew war
-```
-
-By default `try.kotl.in` uses port 8080, and if that's OK, just make it available at `http://localhost:8080` via this command:
-
-```bash
-$ docker-compose up
-```
-
-To change the port number, tweak 'docker-compose.yml':
-
-```bash
-    ports:
-      - "your_port:8080"
-```
-
-If you'd like to log in to JetBrains account, Google, Facebook, GitHub, or Twitter, add corresponding keys to
-this [configuration file](https://github.com/JetBrains/kotlin-web-demo/blob/master/docker/frontend/conf/Catalina/localhost/ROOT.xml):
-
-```xml
-    <Environment name="github_key" value="YOUR-KEY" type="java.lang.String" override="false"/>
-    <Environment name="github_secret" value="YOUR-SECRET-KEY" type="java.lang.String" override="false"/>
-```
+[![Build Status](https://api.travis-ci.org/47deg/try.arrow-kt.io.svg?branch=master)](https://travis-ci.org/47deg/try.arrow-kt.io)
+
+This repository contains the scripts and necessary files to deploy [Try Arrow](https://try.arrow-kt.io:80) in an AWS EC2 instance.
+
+Try Arrow uses the [kotlin-web-demo](https://github.com/JetBrains/kotlin-web-demo) project and modifies it to include the [Arrow](https://github.com/arrow-kt/arrow) library and to provide it the ability to be automatically deployed in an AWS EC2 instance using [Travis CI](https://travis-ci.org). 
+
+## Project structure:
+
+- The main scripts that do all the "hard work" are `setup.sh` and `deploy.sh`:
+  - The `setup.sh` script is intended to install in the instance all the necessary software (docker, docker-compose, gradle, ...) to deploy the projec. It will have to be run only once. 
+  - The `deploy.sh` script will be run every time a new deployment is triggered and will take care of download the required version of Try Kotlin, modify it, compile the project, clean up the instance and, using docker-compose, build and start the containers. 
+- `arrow` directory contains the following files:
+  - `arrow-dependencies`: Includes the Arrow library's dependencies that will be inserted in the `build.gradle` files of each Kotlin's compiler version.
+  - `arrow-executors-policy`: Contains some Java Security Policies needed to run code from the Arrow docs and will be included in the `executors.policy.template` file. 
+  - `arrow-repositories`: The Arrow repositories to download the library from. This will be also inserte in all `build.gradle` files inside Kotlin's compiler versions.
```

Comment:
```suggestion
  - `arrow-repositories`: The Arrow repositories where to download the libraries. This will be also inserted in all `build.gradle` files inside Kotlin's compiler versions.
```